content/ja/docs/tasks/tools/install-minikube.md

```diff
@@ -49,7 +49,7 @@ MinikubeはVMの中ではなくホスト上のKubernetesのコンポーネント
 [Homebrew](https://brew.sh)を使うことでmacOSにMinikubeを簡単にインストールできます。
 
 ```shell
-brew cask install minikube
```

Comment:
Please remove this change as it's already dealt with in the Japanese translation branch.